containers/Dockerfile.intel-oneapi-ufs-s2s-coupled-lite

```diff
@@ -0,0 +1,14 @@
+FROM intel-oneapi-ufs-s2s-coupled-dev:ubuntu20 as ufs
```

Comment:
what is the use case of this dockerfile?

Reply:
In order to reduce the size of the containers, it's common to delete all the source code, but if one simply removes it in the same container image where it was created, it doesn't reduce the size (hence the need for a multi-stage container build).  It really depends on how one plans to use the container, but a smaller container is easier to move around in general and loads up faster when bringing up a cloud-based master instance.